manifests/compute/foreman_compute.pp

```diff
@@ -0,0 +1,13 @@
+# Provides a common class to install foreman-compute
+# used by multiple compute resources
+#
+# === Parameters:
+#
+# $version::  Package version to install, defaults to installed
+#
+class foreman::compute::foreman_compute ( $version = 'installed' ) {
```

Comment:
Ah yes, fair point, as the other classes don't/can't pass that parameter through.